From 902f10e6dbd27705d224eff10a99083869609c06 Mon Sep 17 00:00:00 2001
From: Nathalie Chan King Choy <nathalie.chan-king-choy@xilinx.com>
Date: Wed, 23 Nov 2016 18:55:20 -0800
Subject: [PATCH 222/223] ARM64: zynqmp: usb1 on zcu100 is not peripheral

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

For the ZCU100 board, usb1 is host and that's specified in dwc3_1.

Signed-off-by: Nathalie Chan King Choy <nathalie@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 2762bc9163bb8576f63ff82801a65576f59e1e57)
---
 arch/arm64/boot/dts/xilinx/zynqmp-zcu100.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu100.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu100.dts
index 364a00c..a67cfdb 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu100.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu100.dts
@@ -462,7 +462,6 @@
 /* ULPI SMSC USB3320 */
 &usb1 {
 	status = "okay";
-	dr_mode = "peripheral";
 };
 
 &dwc3_1 {
-- 
2.7.4

